src/lang-promql/complete/promql.terms.ts

```diff
@@ -1,54 +1,83 @@
-export const matchOpTerms = ['=', '!=', '=~', '!~'];
-export const binOpTerms = ['^', '*', '/', '%', '+', '-', '==', '>=', '>', '<', '<=', '!=', 'and', 'or', 'unless'];
-export const binOpModifierTerms = ['on', 'ignoring', 'group_left', 'group_right'];
+export const matchOpTerms = [{ label: '=' }, { label: '!=' }, { label: '=~' }, { label: '!~' }];
+export const binOpTerms = [
+  { label: '^' },
+  { label: '*' },
+  { label: '/' },
+  { label: '%' },
+  { label: '+' },
+  { label: '-' },
+  { label: '==' },
+  { label: '>=' },
+  { label: '>' },
+  { label: '<' },
+  { label: '<=' },
+  { label: '!=' },
+  { label: 'and' },
+  { label: 'or' },
+  { label: 'unless' },
+];
+export const binOpModifierTerms = [{ label: 'on' }, { label: 'ignoring' }, { label: 'group_left' }, { label: 'group_right' }];
 export const functionIdentifierTerms = [
```

Comment:
> And I don't know yet what short details we should put for the different function / aggregation method

@Nexucis I think it would be great if we can show the first line of the function description from the doc, such as  https://github.com/prometheus/prometheus/blob/e17d18a0433a78b204dff7c3dbf33ff216471ed9/docs/querying/functions.md#changes

I don't know how @juliusv does that in PromLens but I think it's available in their "Explain" tab

Reply:
I think a shorter, more dedicated manual text for this one-line description would be better. I have that in PromLens as well, but only in the form-based editor.

Reply:
when you are talking about `manual text`, are you talking about rewriting the description in order to have a shorter one ?

Reply:
Yes, correct. I have that for each function in PromLens, but of course at some point I have to think about what to OSS :laughing:

Reply:
I believe so. The first line of most description in the doc are already good enough for example. Just need to rephrase some of them and we're good.

I'd be more than happy to contribute some manual work like that :D

Reply:
Oh well... if you're going to do it anyway. Here's a freebie:

https://gist.github.com/juliusv/1ab89524dcf53e0dd3e6ad313b681975

Reply:
Oh guys turn out Grafana already did this https://github.com/grafana/grafana/blob/aae25c530873c0db51d987d1a1bfe84aeda0b3b1/public/app/plugins/datasource/prometheus/promql.ts#L16

![Screenshot from 2020-10-01 22-18-14](https://user-images.githubusercontent.com/5953369/94828973-41949880-0434-11eb-9557-0403219e766d.png)

Reply:
Ah right, we could also reuse that, as it has operators, etc. as well. Thanks!

Reply:
yeah I guess so. I will just add the limit suggested by @juliusv and then merge the PR if there is nothing else to change / adapt

Reply:
we have to wait CMN v0.13.0 to see the description of the function / metrics so it's ok if it's not done in this PR but in another.
The struct is there at least to handle it

Reply:
note: in the gist of @juliusv it's missing the description of the aggregation operator (like sum / avg)

Reply:
so feel free to open a PR @dvkndn whenever it fits you.